plugins/module_utils/_api_data.py

```diff
@@ -3829,6 +3829,30 @@ def join_path(path):
             },
         ),
     ),
+    ('queue', 'simple'): APIData(
+        unversioned=VersionedAPIData(
+            primary_keys=('name', ),
+            fully_understood=True,
+            fields={
+                'bucket-size': KeyInfo(default='0.1'),
+                'burst-limit': KeyInfo(default=0),
+                'burst-threshold': KeyInfo(default=0),
+                'burst-time': KeyInfo(default='0s'),
+                'comment': KeyInfo(can_disable=True, remove_value=''),
+                'disabled': KeyInfo(default=False),
```

Comment:
I am not sure if setting the default for `disabled` to `False` is practical here. Even when setting it to `no` directly on a device it is neither shown in the output of `/queue simple export` nor in the output of `/queue simple print detail`.
Setting the default to `False` here looks like it would lead to changes on every execution. Can you confirm please?

Reply:
I can confirm that setting to the disabled default to 'False' does not result in a change every time.

You're correct in that it does not appear in /export or api_info, I can remove this default if you like.

```
k: [60brh-dc2-rtr1] => {
    "show_interface_queues": {
        "changed": false,
        "failed": false,
        "result": [
            {
                "!dst": null,
                "!time": null,
                ".id": "*1",
                "bucket-size": "0.1/0.1",
                "burst-limit": "0/0",
                "burst-threshold": "0/0",
                "burst-time": "0s/0s",
                "comment": "Core: 172msa-dc1-core1:bonding1-vlan318 {HU-xxx-xxx} [1G]",
                "limit-at": "0/0",
                "max-limit": "0/1000000000",
                "name": "queue-bonding1-vlan318",
                "priority": "8/8",
                "queue": "default-small/default-small",
                "target": "bonding1-vlan318"
            }
        ]
    }
}
```

Reply:
Thank you for testing. If it doesn't lead to constant changes I am fine with it :+1: